### DISCUSSION TITLE
Some more hints on the Contact page

### DISCUSSION THREAD

**Discussion by nkomarov** [Automattic/jetpack#32645] [Ideas]
There is a [Contact Support](https://jetpackme.wordpress.com/contact-support/?rel=support) page where every free user is stuck when he/she has a problem. I understand, there are no resources for unpaid users.

In my case, I see a bug and want to report it. How do I do that? It's not said there. They could at least advise to check the Known issues. They could add a link to the GitHub Issues page. I had to search for that manually. So, the suggestion is just to add some links.

**Comment by jeherve** [comment]
Thanks for the feedback! Free users of the plugin do indeed have limited options. They should not be stuck though!

Instead of suggesting GitHub, which can be a bit overwhelming for non-technical folks (who often will not even have an account on the platform), we've opted to direct folks to the WordPress.org plugin support forums instead. We monitor the plugin forums and reply to everyone reporting issues there. Our support team then reports bugs to GitHub when necessary. 

I believe that's a good alternative, and something that's more accessible than GitHub for most folks. A lot more site owners will have a WordPress.org account than a GitHub account I think.

That said, I can see how surfacing our GitHub repository a bit more could be helpful for more technical folks like you. I'll see that we do that.

Thanks again for bringing this up!

**Reply by jeherve** [reply]
> I can see how surfacing our GitHub repository a bit more could be helpful for more technical folks like you. I'll see that we do that.

I've now added a link to the GitHub repository in the footer on all pages of jetpack.com. Hopefully this helps a bit. I'll see if we can add more links later on if necessary.
